src/core/src/sys/compute.rs

```diff
@@ -236,8 +236,8 @@ impl Compute for PressureAtTemperature {
         let virial_tensor = system.virial();
         let virial = virial_tensor.trace();
         let volume = system.volume();
-        let natoms = system.size() as f64;
-        return natoms * K_BOLTZMANN * self.temperature / volume + virial / (3.0 * volume);
+        let nmols = system.molecules().len() as f64;
```

Comment:
Mhm I did the derivation for non-rigid molecules. It is also completely derived in Tuckerman's Statistical Mechanics (in the solutions to the exercises, I can send you those). From what I can see, Towhee uses the number of molecules exclusively. Is Towhee restricted to rigid molecules?

I don't have Frenkel & Smit here, I will have a look on monday.